docs/rules/jsx-sort-props.md

```diff
@@ -75,6 +76,24 @@ When `true`, alphabetical order is not enforced:
 <Hello tel={5555555} name="John" />
 ```
 
+### `reservedFirst`
+
+This can be a boolean or an array option.
+
+When `reservedFirst` is defined, React reserved props (`children`, `dangerouslySetInnerHTML`, `key`, `ref`) must be listed before all other props, but still respecting the alphabetical order:
```

Comment:
I also don't see any tests that exercise this prop - please add tests that cover every "reserved prop" :-)